cmake/3rdparty.cmake

```diff
@@ -14,7 +14,7 @@ EXTERNALPROJECT_ADD(
     LOG_INSTALL ON
     LOG_CONFIGURE ON
     GIT_REPOSITORY ${STDTRACER_GIT_URL}
-    GIT_TAG v0.2.1
+    GIT_TAG master
```

Comment:
doesn't v0.2.1 work?

Reply:
v0.2.1 doesn't work. It will produce errors like https://github.com/tensorlayer/hyperpose/issues/277

Reply:
Well. This is strange. I retried v0.2.1 and now it works.

Reply:
I think we should figure out why, the current master of stdtracer is v0.2.1, they should be equivalent.

Reply:
After my execution on `make`, there's only 2 folders in the `3rdparty` folder: 

1. src
2. tmp

`include`'s not found.

Reply:
Could you try the commit ID `3426d52a87ea7e35495401cce777147a2eefe69e`

Reply:
Could you also try remove the 3rdparty folder and then rebuild?

Reply:
So strange. It was not performing the downloading step. This seems not to be a problem with the version tag but cmake.EXTERNALPROJECT_ADD.

Reply:
it won't download if one already exists. I guess if you have an old clone, 3rdparty folder won't be updated.

Reply:
I succeeded in installing the 3rdparty by using cmake from pypi. (version == 3.17.3) I don't know if this is a cmake version problem. Anyway let's accept the change with your modification.

Reply:
I succeeded in installing the 3rdparty by using cmake from pypi. (version == 3.17.3) I don't know if this is a cmake version problem. Anyway let's accept the change with your modification.